src/main/resources/org/jenkinsci/plugins/ccxml/CCXMLAction/index.jelly

```diff
@@ -28,19 +28,21 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
-    <st:contentType value="text/xml;charset=UTF-8" />
+    <st:contentType value="application/xml;charset=UTF-8" />
```

Comment:
According to RFC referenced in https://stackoverflow.com/questions/4832357/whats-the-difference-between-text-xml-vs-application-xml-for-webservice-respons . I would argue that "text/xml" was correct since the output is really "readable by casual users". It is also not related to the bug from what I see

Reply:
You're right, I changed it to make the testing logic simpler as `JenkinsRule#createWebClient#goToXml` requires `application/xml`. I will change it back and update the tests.